shared/modules/selectors/networks.ts

```diff
@@ -81,13 +81,13 @@ export const getProviderConfig = createSelector(
         }
       }
     }
-    return undefined; // should not be reachable
+    throw new Error('Provider configuration not found');
```

Comment:
One of two logci changes: if this _is_ reachable, we should yell about it so we know there is a bug.